src/_includes/components/site-nav.html

```diff
@@ -1,3 +1,3 @@
-<nav>
+<nav id="main-nav">
```

Comment:
What is the reasoning for using an id here? 

`.site-nav` would be more appropriate of a selector. It is site navigation. `main-nav` is contextual not semantic.